mava/systems/tf/mad4pg/system.py

```diff
@@ -72,7 +77,6 @@ def __init__(
         bootstrap_n: int = 10,
         sigma: float = 0.3,
         max_gradient_norm: float = None,
-        max_executor_steps: int = None,
```

Comment:
Good point. I think we do need it. However I will code in the logic manually and therefore we have more freedom of choice. So let me know what you want. The system can exit after a certain amount of executor steps, trainer steps or time? Your call.

Reply:
Thanks @DriesSmit ! When you say manually, do you mean without using `lp_utils.StepsLimiter` ?

 I think we can make this flexible, maybe `termination_condition={"executor_steps": 10000}`, that way we can do `termination_condition={"trainer_steps": 10000}` as well.  `StepsLimiter` accepts a `steps_key` key, so that can be the dict key.

Reply:
Done:) You can now use any of the following arguments as termination conditions: trainer_steps, trainer_walltime, evaluator_steps, evaluator_episodes, executor_episodes or executor_steps.